repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -736,9 +749,14 @@ export interface CapDetails {
 }
 
 export interface CapInfo {
+    id?: number;
     cap: Cap;
     capStart?: CapStartInfo;
 }
+
+export interface CapSelection {
```

Comment:
This interface looks to only be extended by CapsDefinition - does it need to be its own interface?

Reply:
yes for the matching json